README.md

```diff
@@ -26,14 +26,76 @@ would be a nice experience.
 Of course, in order to understand the Dockerfile, one might need some introduction.
 So we tried to document each line in the Dockfile for you.
 
+## Getting Started
 
+### Prerequisites
 
+In order to use this repo, you should install:
 
-# Build
+* Docker : For building and run the docker image
+* git: For cloning the repo
+* git-lfs : This is needed because ACS repo keeps larger files that are served with this system. Hopefully, we will remove this in the future.
 
-    docker build . --tag=acs:2020.4
 
-# Run
+Once you have the aforementioned prerequisites, clone the repo like this:
 
-    docker run -dP --name=acs acs:2020.4
+```
+git clone --recursive https://github.com/dneise/acs_test acs_docker
+```
 
+NOTE: This command should **also** clone the ACS repo. If this not happen, once the ACS Docker repo is cloned, checkout the develop branch:
+
+```
+git checkout develop
+```
+
+And update the submodule:
+
+```
+git submodule update --init
+```
+
+### Building the docker image
+
+
+```
+docker build -t alma/acs .
+```
+
+## Deployment
+
+```
+docker run --rm -it --name=acs alma/acs
+```
+
+If you want to compile your ACS module inside the docker container, create a docker volume that binds the path of your machine:
+
+```
+docker run --rm -it -v $PWD:/test -w /test alma/acs
+```
+
+If you need to receive the graphical interface from the container:
+
+```
+docker run --rm -it -u $UID -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix:rw" -v $PWD:/test -w /test alma/acs
+```
+
+## Built With
+
+* Docker
+
+## Contributing
+
+Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
```

Comment:
should we include adding `CONTRIBUTING.md in this PR to make sure we do not end up with an lying README?

Reply:
this way we can also fix #14 with this PR.

Reply:
Agree you can do it

Reply:
okay I need a moment for this ... please do not merge yet

Reply:
done